README.md

```diff
@@ -1,3 +1,147 @@
+- ``/genie alert create``: Allow any user to create a new alert.
+- ``/genie alert note``: Adds a note to an existing alert.
+- ``/genie alert close``: Closes an existing alert.
+- ``/genie alert ack``: Acknowledge an existing alert.
+- ``/genie alert unack``: UnAcknowledge an existing alert.
+- ``/genie alert snooze``: Snooze an existing alert for a period of time.
+- ``/genie alert assign``: Assign an existing to a mattermost team member.
+- ``/genie alert own``: Take ownership of an existing alert (assign alert to yourself).
+- ``/genie alert priority``: Set the priority of an existing alert.
+- ``/genie alert list``: Get a list of the existing alerts.
+- ``/genie team list``: Get a list of the existing teams.
+- ``/genie subscription add``: Creates a new subscription for notifications: choose a team and a channel and get notified of the updates in that team. You can subscribe more than one team per channel.
+- ``/genie subscription list``: Show the list of all subscriptions made in all of your channels.
+- ``/genie subscription remove``: Will allow you to remove a subscription. No more notifications from that team will be received.
+
+# Development environment
+
+## Manual installation
+
+*  Download the latest repository release.
+
+### Run the local development environment
+
+* You need to have installed at least node version 15 and maximum version 18. You can download the latest lts version of node for the required operating system here https://nodejs.org/es/download/
+
+### Install dependencies
+* Move to the project directory or execute ``cd`` command to the project directory and execute ``npm install`` with a terminal to download all dependency libraries.
+
+```
+$ npm install
+```
+
+*  Update the environment configuration file. The ``.env`` file must be modified or added to set the environment variables, it must be in the root of the repository.
+
+```
+file: .env
+
+PROJECT=mattermost-opsgenie-app
+PORT=4002
+HOST=http://localhost:4002
+```
+
+Variable definition
+
+- PROJECT: In case of executing the project with Docker using the ``.build.sh`` file, this variable will be used for the name of the container
+- PORT: Port number on which the OpsGenie integration is listening
+- HOST: OpsGenie API usage URL
+
+* Finally, the project must be executed.
+
+```
+$ npm run dev
+```
+
+Or, if you would like to use the Makefile command:
+
+```
+$ make watch
+```
+
+### Run the local development environment with Docker
+
+* You need to have Docker installed. You can find the necessary steps to install Docker for the following operating systems:
+
+[Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+[Mac](https://docs.docker.com/desktop/mac/install/)
+[Windows](https://docs.docker.com/desktop/windows/install/)
+
+* Once you have Docker installed, the next step would be to run the ``make run-server`` command to create the API container and expose it locally or on the server, depending on the case required.
+
+```
+$ make run-server
```

Comment:
This code example isn't needed since it's included in the instructional step above.